db/migrate/20150222172825_create_accounts.rb

```diff
@@ -7,7 +7,11 @@ def change
       t.string :oauth_access_token
       t.string :oauth_refresh_token
       t.datetime :oauth_expires_at
-      t.timestamps
+      t.text :synced_data
+      t.datetime :synced_all_at
+      t.string :email
```

Comment:
I use local attributes for `email` has can be often accessed that's less processing